tests/Phoenix.MarketData.Infrastructure.Tests/generate-coverage-report.sh

```
@@ -0,0 +1,90 @@
+fi
+
+# Create the report directory if it doesn't exist
+mkdir -p "$REPORT_DIR"
+
+# Find the most recent coverage file
+echo "Searching for coverage files..."
+LATEST_COVERAGE_FILE=""
+LATEST_TIMESTAMP=0
+
+# Find the most recent coverage file
+for file in $(find "$TEST_RESULTS_DIR" -name "coverage.cobertura.xml" 2>/dev/null); do
```

Shellcheck (0.10.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)